### ISSUE TITLE
Upgrade the COAR Resource Types to version 2.0

### ISSUE THREAD

**Issue by jdvorak001** [openaire/guidelines-cris-managers#99]
http://vocabularies.coar-repositories.org/documentation/resource_types/

Recording today's discussion with @joschirr : It would be very difficult to coordinate with the releases of the other OpenAIRE Guidelines, so we should proceed on our own here. It is always recommended to use the latest available version.

**Comment by ACz-UniBi** [comment]
In the last months, COAR has refined and published the new vocabulary at https://vocabularies.coar-repositories.org/resource_types/ , v3.0

**Comment by jdvorak001** [comment]
Version 3.0 has dropped 7 terms from the earlier versions:

> The following concepts were removed from this version of the vocabulary: _interview_, _report par_, _contribution to journal_, _internal report_, _other type of report_, _report to funding agency_ and _periodical_. These concepts remains in the earlier versions of the vocabulary, and the PURL URIs continue to resolve.
(the Changelog section at https://vocabularies.coar-repositories.org/resource_types/, last bullet point)

How do we handle these? I can see two options:

1. Take them away. But then we are no longer doing an extension-only maintenance release.
2. Leave them in, listing them as deprecated. So the XML Schema would allow these URLs, and (ideally) the Schematron validation would issue a warning.

**Comment by ACz-UniBi** [comment]
COAR has updated the resource types vocabulary

v2.0 - 2021-02-03 , https://vocabularies.coar-repositories.org/resource_types/2.0/
v3.0 - 2021-07-19 , https://vocabularies.coar-repositories.org/resource_types/3.0/

From the COAR resource types v1.1, which is used by the current guidelines, to COAR v2.0 is no ChangeLog available. It seems to be, that only terms have been added.

From v2.0 to v3.0 is the ChangeLog:
`* The Dataset top concept was expanded to include 13 sub-concepts: aggregated data, clinical trial data, compiled data, encoded data, experimental data, genomic data, geospatial data, laboratory notebook, measurement and test data, observational data, recorded data, simulation data and survey data.
* The Patent top concept was expanded to include 6 sub-concepts: PCT application, design patent, plant patent, plant variety protection, software patent and utility model.
* The Design top concept was expanded to include 2 sub-concepts: industrial design and layout design.
* The following concepts/sub-concepts were added: source code, conference presentation, other periodical, research protocol, peer review, commentary, transcription and trademark.
* The following concepts were removed from this version of the vocabulary: interview, report par, contribution to journal, internal report, other type of report, report to funding agency and periodical. These concepts remains in the earlier versions of the vocabulary, and the PURL URIs continue to resolve.`

I would suggest, that the next release of the Guidelines marked the removed terms as deprecated and the future release will be remove the terms.

The benefit would be that the guidelines remain compatible with v1.1 and v1.1.1 .

**Comment by jdvorak001** [comment]
Yes, I very much agree we should keep the terms removed in the 2.0 -> 3.0 step: we'd lose backwards compatibility otherwise. In the case of "periodical" and "contribution to journal" this will even involve keeping parts of the old hierarchy.

**Comment by jdvorak001** [comment]
Oops! Version 3.0 uses `http://purl.org/coar/resource_type/scheme` as the scheme URI, whereas for 2.0 and earlier it is just `http://purl.org/coar/resource_type`. This propagates as the XML namespace URI for the XML Schema, so it is introducing an incompatibility. We need to discuss how to patch this crack.

**Comment by ACz-UniBi** [comment]
Dear @jdvorak001 , could you link to page. I see:

v1.1
![image](https://user-images.githubusercontent.com/41193035/154360506-5840e0dc-1286-46fc-85b5-9ea13a9e9a2d.png)

v2.0
![image](https://user-images.githubusercontent.com/41193035/154360579-241a2047-6ea8-491e-a708-d3b3c7724a44.png)

v3.0
![image](https://user-images.githubusercontent.com/41193035/154360663-762a1673-2df3-4ec6-b6e3-219fd8469777.png)

and the namespace URI schema doesn't change it's still "**http://purl.org/coar/resource_type/**"

**Comment by jdvorak001** [comment]
Yes, it's been declared this way. But the skos:ConceptScheme in v3.0 is `http://purl.org/coar/resource_type/scheme`, see [here](https://github.com/jdvorak001/guidelines-cris-managers/blob/coar_vocab_v3/schemas/vocabularies/00-preparations/coar-resource-types-v3.0.xml#L1945). And `http://purl.org/coar/resource_type` is not used anywhere in the file (which is a translation of the [official N-Tripples file](https://vocabularies.coar-repositories.org/resource_types/3.0/resource_types.nt) into RDF/XML).

**Comment by jdvorak001** [comment]
We agreed to keep the deprecated terms in the vocabulary even if we are producing version 1.2.0 (instead of the originally planned 1.1.2). We will drop the date a term is deprecated since, because these were misleading: no new version of the OpenAIRE CRIS Guidelines was produced on that day. We will also add a notice about the possibility to drop these terms in a next release.
